src/libs/ajax/leonardo/LeoRuntimesV1DataClient.ts

```diff
@@ -0,0 +1,40 @@
+import { AbortOption, FetchFn } from 'src/libs/ajax/data-client-common';
+import { RawGetRuntimeItem } from 'src/libs/ajax/leonardo/models/runtime-models';
+
+export interface LeoRuntimesV1DataClient {
+  details: (project: string, name: string, options?: AbortOption) => Promise<RawGetRuntimeItem>;
```

Comment:
These argument names could potentially be a little clearer:

```suggestion
  details: (googleProject: string, runtimeName: string, options?: AbortOption) => Promise<RawGetRuntimeItem>;
```

Reply:
good call, will fix